util/src/test/java/org/killbill/billing/util/tag/api/DefaultTagUserApiTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2014-2015 Groupon, Inc
+ * Copyright 2014-2015 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.util.tag.api;
+
+import org.killbill.billing.callcontext.DefaultCallContext;
+import org.killbill.billing.callcontext.InternalCallContext;
+import org.killbill.billing.util.UtilTestSuiteNoDB;
+import org.killbill.billing.util.api.TagDefinitionApiException;
+import org.killbill.billing.util.callcontext.CallContext;
+import org.killbill.billing.util.callcontext.InternalCallContextFactory;
+import org.killbill.billing.util.tag.TagDefinition;
+import org.killbill.billing.util.tag.dao.TagDao;
+import org.killbill.billing.util.tag.dao.TagDefinitionDao;
+import org.killbill.billing.util.tag.dao.TagDefinitionModelDao;
+import org.mockito.Mockito;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Created by maguero on 14/10/15.
+ */
+public class DefaultTagUserApiTest extends UtilTestSuiteNoDB {
+
+    private DefaultTagUserApi tagUserApi;
+    private TagDefinitionDao tagDefinitionDao;
+    private CallContext context;
+
+    @BeforeMethod
```

Comment:
All TestNG annotations need to specify a test group, either `fast` (when extending `UtilTestSuiteNoDB`, like here) or `slow` (when extending `UtilTestSuiteWithEmbeddedDB`).